sv.c

```diff
@@ -4662,6 +4662,67 @@ Perl_sv_set_undef(pTHX_ SV *sv)
         SvOK_off(sv);
 }
 
+/*
+=for apidoc sv_set_true
+
+Equivalent to C<sv_setsv(sv, &PL_sv_yes)>, but may be made more
+efficient in the future. Doesn't handle set magic.
+
+The perl equivalent is C<$sv = !0;>.
+
+Introduced in perl 5.35.11.
```

Comment:
The 5.35.11 ship has sailed now; so this is inaccurate. We may consider a 5.35.12, or if not we'll have to wait for 5.37 now. :/ I hope it's the former but it is an open discussion.